### ISSUE TITLE
[Youtube] Clip Titles Wrong

### ISSUE THREAD

**Issue by Luke-L** [yt-dlp/yt-dlp#5852]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

Youtube Clips not receiving the name of the clip, instead getting the name of the full video. I have tried using `title`, `clip_title`, and `clip_id`. Does this functionality even exist? 
Here is my example command: `yt-dlp -vU -o "T:\RoosterTeeth\Clips\%(title)s.%(ext)s" https://www.youtube.com/clip/UgkxM6_zYYqsL57atR6OmxQc_m3QLjnedKSZ`

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '-o', 'T:\\RoosterTeeth\\Clips\\%(title)s.%(ext)s', 'https://www.youtube.com/clip/UgkxM6_zYYqsL57atR6OmxQc_m3QLjnedKSZ']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19044-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg git-2020-06-19-2f59946, ffprobe git-2020-06-19-2f59946
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [youtube:clip] Extracting URL: https://www.youtube.com/clip/UgkxM6_zYYqsL57atR6OmxQc_m3QLjnedKSZ
[youtube:clip] UgkxM6_zYYqsL57atR6OmxQc_m3QLjnedKSZ: Downloading webpage
[debug] [youtube] Extracting URL: https://www.youtube.com/watch?v=rfMLH_fPmIk
[youtube] rfMLH_fPmIk: Downloading webpage
[youtube] rfMLH_fPmIk: Downloading android player API JSON
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] UgkxM6_zYYqsL57atR6OmxQc_m3QLjnedKSZ: Downloading 1 format(s): 248+251
[debug] Invoking ffmpeg downloader on "https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=248&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=1673687976&dur=5562.056&lmt=1563308140801420&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgMuedUvKOrrNVjznYDJE1_LB5YXwEu9GsJzoOoI_2QCQCIApqlz22JJ_RfNPd9ZMSWrA4qqV2DUqXpzpr5l5iI8Rf&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D", "https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=251&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=83132666&dur=5562.081&lmt=1563301521102112&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPgrAqFTPTmo3f87YBY4tQWdGMB3AOVx2Mc-1CEB1qWnAiBWeMWB17_qXzbEpM8toH5dzBzFjm5l_ABKRgkNcClPxQ%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D"
[download] Destination: T:\RoosterTeeth\Clips\The Wrong Way to Open a Cereal Box？ - #553 ｜ RT Podcast.webm
[debug] ffmpeg command line: ffmpeg -y -loglevel verbose -headers "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
" -ss 1341.789 -t 60.0 -i "https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=248&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=1673687976&dur=5562.056&lmt=1563308140801420&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgMuedUvKOrrNVjznYDJE1_LB5YXwEu9GsJzoOoI_2QCQCIApqlz22JJ_RfNPd9ZMSWrA4qqV2DUqXpzpr5l5iI8Rf&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D" -headers "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-us,en;q=0.5
Sec-Fetch-Mode: navigate
" -ss 1341.789 -t 60.0 -i "https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=251&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=83132666&dur=5562.081&lmt=1563301521102112&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPgrAqFTPTmo3f87YBY4tQWdGMB3AOVx2Mc-1CEB1qWnAiBWeMWB17_qXzbEpM8toH5dzBzFjm5l_ABKRgkNcClPxQ%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D" -c copy -map "0:0" -map "1:0" -f webm "file:T:\RoosterTeeth\Clips\The Wrong Way to Open a Cereal Box？ - #553 ｜ RT Podcast.webm.part"
ffmpeg version git-2020-06-19-2f59946 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 9.3.1 (GCC) 20200523
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libdav1d --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libsrt --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-libxvid --enable-libaom --disable-w32threads --enable-libmfx --enable-ffnvcodec --enable-cuda-llvm --enable-cuvid --enable-d3d11va --enable-nvenc --enable-nvdec --enable-dxva2 --enable-avisynth --enable-libopenmpt --enable-amf
  libavutil      56. 55.100 / 56. 55.100
  libavcodec     58. 93.100 / 58. 93.100
  libavformat    58. 47.100 / 58. 47.100
  libavdevice    58. 11.100 / 58. 11.100
  libavfilter     7. 86.100 /  7. 86.100
  libswscale      5.  8.100 /  5.  8.100
  libswresample   3.  8.100 /  3.  8.100
  libpostproc    55.  8.100 / 55.  8.100
[tcp @ 0000013c53cd4ec0] Starting connection attempt to 2607:f8b0:4000:44::7 port 443
[tcp @ 0000013c53cd4ec0] Successfully connected to 2607:f8b0:4000:44::7 port 443
[tcp @ 0000013c53ccf880] Starting connection attempt to 2607:f8b0:4000:44::7 port 443
[tcp @ 0000013c53ccf880] Successfully connected to 2607:f8b0:4000:44::7 port 443
Input #0, matroska,webm, from 'https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=248&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=1673687976&dur=5562.056&lmt=1563308140801420&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgMuedUvKOrrNVjznYDJE1_LB5YXwEu9GsJzoOoI_2QCQCIApqlz22JJ_RfNPd9ZMSWrA4qqV2DUqXpzpr5l5iI8Rf&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D':
  Metadata:
    encoder         : google/video-file
  Duration: 01:32:42.06, start: 0.000000, bitrate: 2407 kb/s
    Stream #0:0(eng): Video: vp9 (Profile 0), 1 reference frame, yuv420p(tv, bt709), 1920x1080, SAR 1:1 DAR 16:9, 29.97 fps, 29.97 tbr, 1k tbn, 1k tbc (default)
[tcp @ 0000013c54294fc0] Starting connection attempt to 2607:f8b0:4000:44::7 port 443
[tcp @ 0000013c54294fc0] Successfully connected to 2607:f8b0:4000:44::7 port 443
[tcp @ 0000013c53d7da80] Starting connection attempt to 2607:f8b0:4000:44::7 port 443
[tcp @ 0000013c53d7da80] Successfully connected to 2607:f8b0:4000:44::7 port 443
Input #1, matroska,webm, from 'https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=251&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=83132666&dur=5562.081&lmt=1563301521102112&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPgrAqFTPTmo3f87YBY4tQWdGMB3AOVx2Mc-1CEB1qWnAiBWeMWB17_qXzbEpM8toH5dzBzFjm5l_ABKRgkNcClPxQ%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D':
  Metadata:
    encoder         : google/video-file
  Duration: 01:32:42.08, start: -0.007000, bitrate: 119 kb/s
    Stream #1:0(eng): Audio: opus, 48000 Hz, stereo, fltp, delay 312 (default)
Output #0, webm, to 'file:T:\RoosterTeeth\Clips\The Wrong Way to Open a Cereal Box？ - #553 ｜ RT Podcast.webm.part':
  Metadata:
    encoder         : Lavf58.47.100
    Stream #0:0(eng): Video: vp9 (Profile 0), 1 reference frame, yuv420p(tv, bt709), 1920x1080 (0x0) [SAR 1:1 DAR 16:9], q=2-31, 29.97 fps, 29.97 tbr, 1k tbn, 1k tbc (default)
    Stream #0:1(eng): Audio: opus, 48000 Hz, stereo, fltp, delay 312 (default)
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
  Stream #1:0 -> #0:1 (copy)
Press [q] to stop, [?] for help
Automatically inserted bitstream filter 'vp9_superframe'; args=''
No more output streams to write to, finishing.e=00:00:58.54 bitrate=2435.8kbits/s speed=1.99x
frame= 1852 fps= 62 q=-1.0 Lsize=   18102kB time=00:00:59.99 bitrate=2471.9kbits/s speed=2.01x
video:17242kB audio:825kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.193691%
Input file #0 (https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=248&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=video%2Fwebm&gir=yes&clen=1673687976&dur=5562.056&lmt=1563308140801420&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRAIgMuedUvKOrrNVjznYDJE1_LB5YXwEu9GsJzoOoI_2QCQCIApqlz22JJ_RfNPd9ZMSWrA4qqV2DUqXpzpr5l5iI8Rf&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D):
  Input stream #0:0 (video): 1853 packets read (17659134 bytes);
  Total: 1853 packets (17659134 bytes) demuxed
Input file #1 (https://rr2---sn-q4flrnes.googlevideo.com/videoplayback?expire=1671754354&ei=Ep6kY-SCDLSU2_gPqsqggAk&ip=2603%3A8080%3Aa00%3A9fd2%3A53%3Abfcb%3A5d62%3Aa386&id=o-AAX9qsfxXk9x72ePu88fy6hx1N5fzMntnz3QhxtB6L7Z&itag=251&source=youtube&requiressl=yes&mh=AI&mm=31%2C26&mn=sn-q4flrnes%2Csn-a5msenl7&ms=au%2Conr&mv=m&mvi=2&pl=35&initcwndbps=1857500&spc=zIddbMitVcWVob4r3M52qbvHwdASRVo&vprv=1&svpuc=1&mime=audio%2Fwebm&gir=yes&clen=83132666&dur=5562.081&lmt=1563301521102112&mt=1671732572&fvip=4&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cspc%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIhAPgrAqFTPTmo3f87YBY4tQWdGMB3AOVx2Mc-1CEB1qWnAiBWeMWB17_qXzbEpM8toH5dzBzFjm5l_ABKRgkNcClPxQ%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIgez-Rw6Qx4kpZyRSs00HwR2IZjjUjzPuuFJlESiXURngCIQC5e2V4uF5QmVcGENnOYIfrVmqqM--9st-O6TgWMldZyw%3D%3D):
  Input stream #1:0 (audio): 3090 packets read (844604 bytes);
  Total: 3090 packets (844604 bytes) demuxed
Output file #0 (file:T:\RoosterTeeth\Clips\The Wrong Way to Open a Cereal Box？ - #553 ｜ RT Podcast.webm.part):
  Output stream #0:0 (video): 1852 packets muxed (17656231 bytes);
  Output stream #0:1 (audio): 3089 packets muxed (844394 bytes);
  Total: 4941 packets (18500625 bytes) muxed
[AVIOContext @ 0000013c53d7d940] Statistics: 2 seeks, 72 writeouts
[AVIOContext @ 0000013c542892c0] Statistics: 17728690 bytes read, 1 seeks
[AVIOContext @ 0000013c53d8e480] Statistics: 901120 bytes read, 1 seeks
[ffmpeg] Downloaded 18536459 bytes
[download] 100% of   17.68MiB in 00:00:31 at 583.49KiB/s
```

**Comment by Marco-9456** [comment]
Any updates on this issue?

**Comment by sabihismail** [comment]
+1 🙏

**Comment by dirkf** [comment]
OP's problem was not liking the title of the extracted clip.

AIUI, the clip extractor `youtube:clip` only extracts **one** clip and gives it the title of the video from which the clip was pulled. If someone didn't like that, the output template `title` field could be modified to add ` (clip)`, say, or perhaps `section_start/end` data, which would disambiguate videos extracted if the clip page could have different in-out points over time.

The scenario [suggested by @bagel-deficient](https://github.com/yt-dlp/yt-dlp/issues/5852#issuecomment-2731925425) is different because it could only occur where multiple clips are being extracted and so can't apply to the clip extractor: it needs a new issue to document the problem fully, but only after considering why you might override the carefully selected default output template with one that fails?

Also:
1. the problem URL is no longer supported by `youtube:clip`: it's a `twoColumnBrowseResultsRenderer` playlist with no `currentVideoEndpoint` but `suitable()` can't know that - does this need investigating?
2. there's no point trying to use `clip_id` in the output template because that becomes the `id` (that you shouldn't omit, as above) for the clip video.
